tensorflow/python/kernel_tests/rnn_cell_test.py

```diff
@@ -105,6 +105,53 @@ def testBasicLSTMCell(self):
                                     m.name: 0.1 * np.ones([1, 4])})
         self.assertEqual(len(res), 2)
 
+  def testLSTMCellBlock(self):
+    with self.test_session() as sess:
+      with tf.variable_scope("root", initializer=tf.constant_initializer(0.5)):
+        x = tf.zeros([1, 2])
+        m = tf.zeros([1, 8 * 7])
+        g, out_m = tf.nn.rnn_cell.MultiRNNCell(
```

Comment:
it does that in the rnn_test.py , or did you want an explicit one for
rnn_cell_test as well?

--
William Chan
Carnegie Mellon University
(650) 450-9455
williamchan.ca

On Tue, Apr 19, 2016 at 4:13 PM, ebrevdo <notifications@github.com> wrote:

> In tensorflow/python/kernel_tests/rnn_cell_test.py
> <https://github.com/tensorflow/tensorflow/pull/2002#discussion_r60328158>:
>
> > @@ -105,6 +105,53 @@ def testBasicLSTMCell(self):
> >                                      m.name: 0.1 * np.ones([1, 4])})
> >          self.assertEqual(len(res), 2)
> >
> > +  def testLSTMCellBlock(self):
> > +    with self.test_session() as sess:
> > +      with tf.variable_scope("root", initializer=tf.constant_initializer(0.5)):
> > +        x = tf.zeros([1, 2])
> > +        m = tf.zeros([1, 8 * 7])
> > +        g, out_m = tf.nn.rnn_cell.MultiRNNCell(
>
> Can I convince you to test against the BasicLSTMCell for both fw and
> backprop w.r.t. inputs & variables?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2002/files/c02daba837c145354bfc75592b25c715d6ab4f1b#r60328158>
>